tests/unit/test_notebooks_python.py

```diff
@@ -87,6 +87,19 @@ def test_wikidata_runs(notebooks, tmp):
         ),
     )
 
+@pytest.mark.notebooks
```

Comment:
how long does this test take? if it takes too long (ex. more than a couple of minutes) it might be better to move it to smoke